packages/next/cms/components/ProfileRenderer.js

```diff
@@ -4,462 +4,68 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable react/jsx-props-no-spreading */
 /* eslint-disable no-nested-ternary */
-/*
- TODO:
-   - [ ] port remaining sections/code
-   - [ ] port styles to Mantine/theme
-   - [ ] User Re-auth/populate
-   - [ ] CustomSections
-   - [ ] test/enable comparison mode
-   - [ ] test/enable PDF printing
-*/
 
-import React, {useEffect, useMemo, useState} from "react";
-// import axios from "axios";
+import React, {useState} from "react";
 import {useRouter} from "next/router.js";
-import {assign} from "d3plus-common";
 import {Container} from "@mantine/core";
-// eslint-disable-next-line import/no-cycle
-import {Hero, ProfileContext} from "../..";
-
-// import {Button, Dialog, Icon} from "@blueprintjs/core";
+import {Hero, NonIdealState, ProfileContext} from "../..";
+import {useProfileSections, useReportState, useOnSelector, useOnSetVariables, useOnTabSelect} from "../hooks";
 
 import Section from "./sections/Section";
 import Related from "./sections/Related";
 import SectionGrouping from "./sections/components/SectionGrouping";
 import Subnav from "./sections/components/Subnav";
 import Mirror from "./Viz/Mirror";
-import mortarEval from "../utils/mortarEval";
-import deepClone from "../utils/deepClone";
-import prepareProfile from "../utils/prepareProfile";
-import funcifyFormatterByLocale from "../utils/funcifyFormatterByLocale";
 
-// TODO
-// User must define custom sections in app/cms/sections, and export them from an index.js in that folder.
-// import * as CustomSections from "CustomSections";
+import ComparisonButton from "./fields/ComparisonButton";
 
-// TODO
-// const comparisonsEnabled = process.env.NEXT_PUBLIC_PROFILE_COMPARISON;
+const comparisonsEnabled = process.env.NEXT_PUBLIC_PROFILE_COMPARISON;
 // const comparisonExclude = process.env.NEXT_PUBLIC_PROFILE_COMPARISON_EXCLUDE;
```

Comment:
I know of the exact use case for this env var. I didn't approve it's inclusion in canon-cms, and I would have probably done it as a React prop or a CSS hack in the project, but InsightsCloud has this env var:

```sh
export CANON_CONST_PROFILE_COMPARISON_EXCLUDE="skills_and_occupation"
```
What this is doing is saying "don't show the comparison button for the following comma-separated profile slugs". As an example, `"country,hs"` could also be a valid value for a site like the OEC.

Might as well just include, shouldn't be too hard, right?

Reply:
Test with
```
NEXT_PUBLIC_TESSERACT=https://monshaat.api.dev.datawheel.us/tesseract/
NEXT_PUBLIC_CMS=https://monshaat.cms.dev.datawheel.us/
NEXT_PUBLIC_PROFILE_COMPARISON=true
NEXT_PUBLIC_PROFILE_COMPARISON_EXCLUDE="manufacturing"
```

Reply:
@scespinoza not quite the logic here, the exclusion is the _entire profile type_, not for specific members within. It's not something we would even really use right now in Monsha'at, because currently there is only 1 profile type in that project ("sector").

As an example, if a site had two different profiles, one for Countries (`/profile/country/<memberSlug>`) and another for Products (`/profile/hs/<memberSlug>`), and we wanted to enable comparisons for all current and future profile types _except_ for product profiles, we would set:

```sh
NEXT_PUBLIC_PROFILE_COMPARISON_EXCLUDE="hs"
```

It's the slug of the primary profile _dimension_, not an individual _member_ of that dimension.

Reply:
fixed